source/Calamari.Tests.Shared/Calamari.Tests.Shared.csproj

```diff
@@ -14,7 +14,7 @@
 
     <ItemGroup>
         <PackageReference Include="Assent" Version="1.6.1" />
-        <PackageReference Include="Calamari.Common" Version="19.2.5" />
```

Comment:
😬

Reply:
? Concerned about minor patch? Or just that this reference was left behind for a while?

Reply:
Yeah the latter, it is nice we are bringing it back in line :)